Cargo.toml

```diff
@@ -155,6 +155,7 @@ bench = false                               # TODO fix this benchmark
 required-features = ["parquet_compression"]
 
 [package.metadata.docs.rs]
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
To see what this gets us, you can run:

```shell
RUSTDOCFLAGS="--cfg docsrs" cargo +nightly doc --all-features -F gdal/bindgen
```

That will get us feature flags in the output, which is how docs.rs presents stuff. E.g. on the `io` mod:

![image](https://github.com/user-attachments/assets/c7dd16ce-5264-4717-bf96-3c62516975f3)

Reply:
Oh I've been wondering how to get the feature flag badges in the docs! That looks great!